### ISSUE TITLE
External Diff should escape paths

### ISSUE THREAD

**Issue by iSuper-Ray** [ForkIssues/TrackerWin#1345]
I have an issue with an external diff on Windows. This is a new issue in 1.67. I have a space in my user name.
Please check the command log from the external diff command:
```
$ git -c core.longpaths=true --work-tree=C:\Users\Vitaly Uvarov\AppData\Local\Temp\Fork\10e7e70~ restore --source=10e7e700ed23b77ef090f1a1a02e7e268b87be63~ -- DBScripts/QA/sm_172.0.0300.0-MonitorAvailabilityGroup.IgnoreKnownIssue.sql

git: 'Uvarov\AppData\Local\Temp\Fork\10e7e70~' is not a git command. See 'git --help'.
```

All path parameters should be escaped with quotes. This is true at least for Windows.

The correct command in my case should be:
```
git -c core.longpaths=true --work-tree="C:\Users\Vitaly Uvarov\AppData\Local\Temp\Fork\10e7e70~" restore --source=10e7e700ed23b77ef090f1a1a02e7e268b87be63~ -- "DBScripts/QA/sm_172.0.0300.0-MonitorAvailabilityGroup.IgnoreKnownIssue.sql"
```

**Comment by DanPristupov** [comment]
Please check for updated. I've just released the 1.67.1 hotfix.

**Comment by iSuper-Ray** [comment]
Thank you @DanPristupov ! It works like a charm.
